test/league_stats_test.rb

```diff
@@ -0,0 +1,141 @@
+require_relative 'test_helper'
+require './lib/league_stats'
+require './lib/stat_tracker'
+
+class LeagueStatsTest < Minitest::Test
+  def setup
+    game_path = './data/games_truncated.csv'
+    team_path = './data/teams.csv'
+    game_teams_path = './data/game_teams_truncated.csv'
+    @locations = {
+                games: game_path,
+                teams: team_path,
+                game_teams: game_teams_path
+              }
+    @stat_tracker = StatTracker.from_csv(@locations)
+    @league_stats = LeagueStats.new(@stat_tracker.games, @stat_tracker.teams, @stat_tracker.game_teams)
+    @game_teams = @league_stats.game_teams[1]
```

Comment:
This line is added to test some helper methods. This may be a candidate for using a mock or stub in the test, perhaps?